src/cljs/athens/views/pages/graph.cljs

```diff
@@ -443,8 +443,8 @@
                                  :links links}
               ;; example data
               #_{:nodes [{"id" "foo", "name" "name1", "val" 1}
-                         {"id" "bar", "name" "name2", "val" 10}]
-                 :links [{"source" "foo", "target" "bar"}]}
+                        {"id" "bar", "name" "name2", "val" 10}]
+                :links [{"source" "foo", "target" "bar"}]}
```

Comment:
I think you should revert this hunk, the alignment is now wrong here. did `cljstyle` like this?